modules/persistent-storage-csi-gcp-hyperdisk-storage-pools.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-gcp-hyperdisk-storage-pools_{context}"]
+= Storage pools for hyperdisk-balanced disks
+
+Hyperdisk storage pools can be used with Compute Engine for large-scale storage. A hyperdisk storage pool is a purchased collection of capacity, throughput, and IOPS, which you can then provision for your applications as needed. You can use hyperdisk storage pools to create and manage disks in pools and use the disks across multiple workloads. By managing disks in aggregate, you can save costs while achieving expected capacity and performance growth. By using only the storage that you need in hyperdisk storage pools, you reduce the complexity of forecasting capacity and reduce management by going from managing hundreds of disks to managing a single storage pool.
```

Comment:
Where do we mention how to use storage pools?
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/docs/kubernetes/user-guides/storage-pools.md#enabling-storage-pools-for-a-manual-deployment-of-gce-pd-csi-driver

Reply:
@gcharot Done. PTAL.